### PR TITLE
subset of updated checksums

### PR DESCRIPTION
**Pull request by St3V0Bay**

### CODE REVIEW

File: datasets/0.8/fashion-mnist/metadata.json [hunk 1]
```diff
@@ -96,4 +96,4 @@
       ]
     }
   ]
-}
+}
```
Comment (marcenacp) [inline datasets/0.8/fashion-mnist/metadata.json:99]: Can you keep the end-of-line?

**Review by marcenacp — CHANGES_REQUESTED** [review]
Thanks a lot for the PR!

File: datasets/0.8/fashion-mnist/metadata.json (hunk 1 above)
Comment (St3V0Bay) [reply datasets/0.8/fashion-mnist/metadata.json:99]: sure sure. that's a bit a tricky one. VSC removes this automatically when saving the file unless I set the following 
Screenshot 2024-02-12 at 20 05 54

Comment (marcenacp) [reply datasets/0.8/fashion-mnist/metadata.json:99]: Yes! Can you please set this option?

**Review by marcenacp — APPROVED** [review]

**Pull request merged**